nautobot/utilities/testing/views.py

```diff
@@ -574,42 +574,29 @@ def test_list_objects_anonymous(self):
 
         @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
         def test_list_objects_filtered(self):
-            instance1, instance2 = self._get_queryset().all()[:2]
+            instance1 = self._get_queryset().first()
             response = self.client.get(f"{self._get_url('list')}?id={instance1.pk}")
             self.assertHttpStatus(response, 200)
             content = extract_page_body(response.content.decode(response.charset))
             # TODO: it'd make test failures more readable if we strip the page headers/footers from the content
-            if hasattr(self.model, "name"):
-                self.assertIn(instance1.name, content, msg=content)
-                self.assertNotIn(instance2.name, content, msg=content)
-            try:
-                self.assertIn(self._get_url("view", instance=instance1), content, msg=content)
-                self.assertNotIn(self._get_url("view", instance=instance2), content, msg=content)
-            except NoReverseMatch:
-                pass
+            # We should have exactly one table row, and that row is "even" (I guess counting from zero?)
+            self.assertIn('<tr class="even">', content, msg=content)
```

Comment:
After removing the above tests does it make sense to count the rows here to make sure we did only get one row back?

Reply:
I'm actually going to mostly revert this set of changes as it's still too fragile (fails for Job table, for example, since it has a custom table HTML template).